Program.cs

```diff
@@ -1,16 +1,161 @@
 ﻿// See https://aka.ms/new-console-template for more information
 
+using System.Globalization;
+using System.Runtime.InteropServices.JavaScript;
 using Kristina_Kulich__Application;
+using Kristina_Kulich__Application.Helpers;
+using Kristina_Kulich__Application.CSharpLinq;
 
 class Program
 {
     static void Main(string[] args)
     {
-     new Homework_1().Task_1();   
-     new Homework_1().Task_2();
-     new Homework_1().Task_3();
-     new Homework_1().Task_4();
-     new Homework_1().Task_5();
-     new Homework_1().Task_6();
-     }
+        #region Task1
+
+        int[] integers = new[] { 12, 8, 27, 78, 2, 3, 3, 4, 21, 1 };
+        List<int> divedsby3 = integers.Select(i => i).Where(i => (i % 3 == 0)).ToList();
+        foreach (var v in divedsby3)
+        {
+            Console.WriteLine("Devides by 3 without remainder: {0}", v);
+        }
+
+        #endregion
+
+        #region Task2
+
+        int[] integers2 = new[] { 15, -8, -7, -5, 6, 14, 13, -9, 11, -10 };
+        List<int> lesser10andnegative = integers2.Where(i => i < 10).ToList();
+        foreach (var v in lesser10andnegative)
+        {
+            Console.WriteLine("Less than 10 and negative: {0}", v);
+        }
+        int[] integers2 = new[] { 15, -8, -7, -5, 6, 14, 13, -9, 11, -10 };
+        List<int> lesser10andnegative = integers2.Where(i => i < 10).ToList();
+        foreach (var v in lesser10andnegative)
+        {
+            Console.WriteLine("Less than 10 and negative: {0}", v);
+        }
+
+        #endregion
+
+        #region Task3
+
+        List<string> brands = new List<string> { "chanel", "dior", "gucci", "stella mccartni", "adidas" };
+        var filteredbrands = from b in brands
+            where Helpers.CheckIfIsDevidedWithoutRamainder(b.Length, 2) ==
+                  true // b.Length== Helpers.CheckIfIsDevidedWithoutRamainder(b.Length, 2)
+            orderby b
+            select b;
+        foreach (var v in filteredbrands)
+        {
+            Console.WriteLine("Brands have even length: {0}", v);
+        }
+
+        #endregion
+
+        #region Task4
+
+        List<string> cities = new List<string>() { "Berlin", "London", "Paris", "Bobruisk", "Tokyo" };
+        var citiesfiltered = cities.Select(i => i).Where(i => i.StartsWith("B") && i.EndsWith("k")).ToList();
```

Comment:
it is better to not hardcode values, use variables instead. Example var startsWithNumber = "B"

Reply:
fixed